### ISSUE TITLE
Notify previous learners of nearby workshop when registrations are open

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/amy#524]
Can we let people in previous workshops opt in to be alerted the next time a workshop is organized in their area, in case they want to be helpers or be learners again? Also can we notify those that opt in by email when the registrations for that workshop is open?

**Comment by pbanaszkiewicz** [comment]
This would probably have to be integrated into Eventbrite registration form.

I believe would look like:

* set up Eventbrite registration page
* get CSV with attendees from EB and use "bulk upload" feature to add many people to the database to specific event (with their consents for workshop notifications)
* have a worker go through events and notify about workshops

The issue here is I don't know if we can extend EB registration to get the consent and then return this field as part of the CSV.

**Comment by maneesha** [comment]
I like exploring ideas to keep learners connected with SWC.  We can explicitly ask for consent on Eventbrite registration -- something like "I consent for Software Carpentry to contact me by email about future news, events, and other information that may be of interest to me."

Some concerns with this:

* Are we assuming an opt-in or opt-out consent policy when someone signs up on Eventbrite? Should we explicitly ask for consent?  In that case it would only apply to those who signed up on our Eventbrite page.
* We don't always manage registration ourselves so we don't always get names & email addresses of all learners  and can't manage this level of consent.
* Registered != Attended
* Just because they attended a workshop in a given location doesn't mean they are local to that location

**Comment by pbanaszkiewicz** [comment]
Ref #763.

**Comment by gvwilson** [comment]
Can @maneesha @jduckles and @tracykteal please figure out what they'd like and then assign this to @pbanaszkiewicz for prioritization/design/implementation?

**Comment by maneesha** [comment]
I don't want to prioritize this.   Just because a workshop is in a certain geographic area it does not mean that it is open to the public.  We also can't easily flag open/closed registration because it's not a binary like that.  Sometimes it's closed for the first two weeks then open to public,  or sometimes it's 40% seats reserved for affiliates, etc.

If we did something like this,  I foresee emails like "Hi, I just got a notification about a workshop at Harvard University but when I go to the website I can't register.  What should I do?"

I don't think we need to do anything more than have  workshops published on the respective SWC or DC websites.
